receiver/gitproviderreceiver/internal/scraper/githubscraper/README.md

```diff
@@ -0,0 +1,119 @@
+# GitHub Limitations
```

Comment:
Maybe add a link to this README in the top level README, where you're referencing the GitHub scraper?